root/static/scripts/edit/URLCleanup.js

```diff
@@ -3561,76 +3561,47 @@ Object.values(LINK_TYPES).forEach(function (linkType) {
-      ),
-      result: false,
-    };
-  }
-  if (/^(https?:\/\/)?([^.\/]+\.)?wikidata\.org\//.test(url)) {
-    return {
-      error: l(
-        `Wikidata is not a discography entry. Please add this Wikidata link
-         to the release group instead.`,
-      ),
-      result: false,
-    };
-  }
-  return discographyRule(url);
-};
-
-// avoid Wikipedia/Wikidata being added as release-level license entry
-const licenseRule = validationRules[LINK_TYPES.license.release];
-validationRules[LINK_TYPES.license.release] = function (url) {
-  if (/^(https?:\/\/)?([^.\/]+\.)?wikipedia\.org\//.test(url)) {
-    return {
-      error: l(
-        `Wikipedia is not a license page. Please add this Wikipedia link
-         to the release group instead.`,
-      ),
-      result: false,
-    };
-  }
-  if (/^(https?:\/\/)?([^.\/]+\.)?wikidata\.org\//.test(url)) {
-    return {
-      error: l(
-        `Wikidata is not a license page. Please add this Wikidata link
-         to the release group instead.`,
-      ),
-      result: false,
-    };
-  }
-  return licenseRule(url);
-};
+const relationshipTypesByEntityType = Object.entries(LINK_TYPES).reduce(
+  function (
+    results,
+    [/* relationshipType */, relUuidByEntityType],
+  ) {
+    for (const entityType of Object.keys(relUuidByEntityType)) {
+      (results[entityType] || (results[entityType] = []))
+        .push(relUuidByEntityType[entityType]);
+    }
+    return results;
+  },
+  {},
+);
 
-// avoid Wikipedia/Wikidata being added as release-level show notes entry
-validationRules[LINK_TYPES.shownotes.release] = function (url) {
-  if (/^(https?:\/\/)?([^.\/]+\.)?wikipedia\.org\//.test(url)) {
-    return {
-      error: l(
-        `Wikipedia is not a page of show notes. Please add this Wikipedia link
-         to the release group instead.`,
-      ),
-      result: false,
-    };
-  }
-  if (/^(https?:\/\/)?([^.\/]+\.)?wikidata\.org\//.test(url)) {
-    return {
-      error: l(
-        `Wikidata is not a page of show notes. Please add this Wikidata link
-         to the release group instead.`,
-      ),
-      result: false,
-    };
-  }
-  return {result: true};
-};
+// Avoid Wikipedia/Wikidata being added as release-level relationship
+for (const relUuid of relationshipTypesByEntityType.release) {
+  const relRule = validationRules[relUuid];
+  validationRules[relUuid] = function (url) {
+    if (/^(https?:\/\/)?([^.\/]+\.)?wikipedia\.org\//.test(url)) {
+      return {
+        error: l(
+          `Wikipedia has no dedicated article per release.
+           Please add this Wikipedia link to the release group instead,
+           if appropriate.`,
+        ),
+        result: false,
+      };
+    }
+    if (/^(https?:\/\/)?([^.\/]+\.)?wikidata\.org\//.test(url)) {
+      return {
+        error: l(
+          `Wikidata has no dedicated item per release.
+           Please add this Wikidata link to the release group instead,
+           if appropriate.`,
```

Comment:
I kinda feel that "Wikidata has no dedicated item per release" is a bit confusing to non-experts. What about something like "Wikidata normally has no entries that map to specific releases, so adding Wikidata links to a release is currently blocked. Please add this Wikidata link to the release group instead, if appropriate." (and same for Wikipedia).

Reply:
Followed your suggestion in commit https://github.com/metabrainz/musicbrainz-server/pull/1881/commits/633930f0ea831022d5a8782ca9f841d0f26e0d25.

Reply:
I specified "normally" because I'm sure there are a few rare exceptions, but we block them anyway because if we allowed them then 99% of the time people would add wrong stuff so it's better to miss out on them. Maybe not a big deal though.

Reply:
Sorry, that was not intended, I just misread it, should be fixed now, see commit https://github.com/metabrainz/musicbrainz-server/commit/8fdde06e862df1aaeef7a5991515e464b9423d87.